did_doc/src/schema/verification_method/verification_method.rs

```diff
@@ -37,9 +38,16 @@ impl VerificationMethod {
         &self.verification_method_type
     }
 
-    pub fn public_key(&self) -> &PublicKeyField {
+    pub fn public_key_field(&self) -> &PublicKeyField {
```

Comment:
To stay consistent with other methods (even across different structs/impls) I'd suggest to keep the original name and come up with alternative name for the new method

Reply:
Remember that `public_key` is not a field of `VerificationMethod`. This is a function which returns a representation of the value of `public_key_*`, and that representation has a type called `PublicKeyField`. Hence, I think this naming is not breaking consistency (which is getter names and the corresponding fields having the same name).

Reply:
Ah I see it now. Make sense